lib/ddsketch-agent/tests/one_thousand_single_points_ddsketch.rs

```diff
@@ -21,8 +21,8 @@ fn test_one_thousand_single_points_ddsketch() {
     let after: MathableHeapStats = dhat::HeapStats::get().into();
 
     let diff = after - before;
-    dhat::assert_eq!(diff.total_blocks, 1426);
-    dhat::assert_eq!(diff.total_bytes, 249802);
+    dhat::assert_eq!(diff.total_blocks, 21);
+    dhat::assert_eq!(diff.total_bytes, 6096);
```

Comment:
This is extra cool because it now beats the allocation counts for batched inserts.